src/pnpq/apt/protocol.py

```diff
@@ -475,6 +539,70 @@ def to_bytes(self) -> bytes:
         )
 
 
+@dataclass(frozen=True, kw_only=True)
+class AptMessageWithDataEncCount(AptMessageWithData):
+    data_length: ClassVar[int] = 14
+
+    # There are some more bytes at the end according to the documentation:
+    # (WORD - channel 2 identifier, LONG, LONG, LONG)
+    # However, it says that these are for future use and should be ignored.
```

Comment:
This comment is a bit confusing. It would be better to explicitly call out that the documentation says the message is 34 bytes long, but the actual message sent by the specific waveplate model we know about is 20 bytes.

"Ignoring" bytes would mean to me that we read 34 bytes and then don't process the extra 14 bytes. But that's not the case. The message is in fact only 20 bytes long. This is a critical difference, since if we failed to read 14 bytes that were actually there, we would not be able to correctly process the next message on the wire either.

Reply:
That's actually exactly what the documentation says (although the documentation itself is somewhat poor)

Reply:
I know that documentation uses the word "ignore." I'm explaining why "ignore" doesn't necessarily mean "don't read."